repos.csv

```diff
@@ -1,12 +1,2 @@
 cloneUrl,branch
-https://github.com/openrewrite/rewrite-cucumber-jvm,main
-https://github.com/openrewrite/rewrite-hibernate,main
 https://github.com/openrewrite/rewrite-javascript,main
```

Comment:
Why did we remove all of the repos here? I think it would be nice to include at least like 3 to show what that looks like.

Reply:
whoops. I was just removing to test. lemme add that back